src/FindStmtCost.h

```diff
@@ -0,0 +1,147 @@
+#ifndef FINDSTMTCOST_H
+#define FINDSTMTCOST_H
+
+#include "Error.h"
+#include "ExternFuncArgument.h"
+#include "Function.h"
+#include "IRVisitor.h"
+#include "Module.h"
+
+#include <stdexcept>
+#include <unordered_map>
+
+using namespace Halide;
+using namespace Internal;
+using namespace std;
+
+#define NORMAL_NODE_CC 1
```

Comment:
Prefer to use either `constexpr int` or `enum` for numeric constants; use #define only when unavoidable

Reply:
And if they can be moved to a single TU, then they should be, too.